feature-libs/cart/import-export/core/services/export.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ExportService {
+  download(
+    textData: string,
+    {
+      fileName = 'data',
+      extension = 'csv',
+      type = 'text/csv;charset=utf-8;',
+      downloadDelay = 0,
+    } = {}
+  ) {
+    setTimeout(() => {
+      const blob = new Blob(['\ufeff' + textData], {
+        type,
+      });
+      const link = document.createElement('a');
+      const url = URL.createObjectURL(blob);
+
+      link.setAttribute('href', url);
+      link.setAttribute('download', `${fileName}.${extension}`);
+      link.style.visibility = 'hidden';
+      document.body.appendChild(link);
+      link.click();
+      document.body.removeChild(link);
+    }, downloadDelay);
+  }
+}
```

Comment:
`dataToCsv` method is missing here

Reply:
This is the intended action. Here (ExportService) we are focusing on just downloading the file, we don't care about .csv format. `dataToCsv` stay in `ExportCsvSerive` https://github.com/SAP/spartacus/pull/13796/files#diff-7d4d3f7352ecc216095d6560930ddca52fff6159b5c4fa772295bb40d9cde643L20